### ISSUE TITLE
Erskine_1996: Processing submission

### ISSUE THREAD

**Issue by ehwenk** [traitecoevo/austraits.build#255]
**Source**: Erskine et al (1996) Water availability - a physiological constraint on nitrate
utilization in plants of Australian semi-arid mulga woodlands Plant, Celt and Environment* http://doi.org/doi-org.wwwproxy1.library.unsw.edu.au/10.1111/j.1365-3040.1996.tb00430.x
**Custodian**: Suzanne Schmidt
**Handled by**: Lizzy Wenk

- [ ] Contact made with contributor (label as `waiting on response` if waiting for files)
- [ ] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [ ] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [ ] Metadata complete, builds into Austraits
- [ ] Report reviewed by AusTraits team
- [ ] Report sent to contributor ( label as `waiting on response` )
- [ ] Response received from contributor
- [ ] Files updated & report finalised
- [ ] Final checks ( all tests pass )
- [ ] Branch merged into master branch (issue `closed` )

NB: 

- if you need to change the config files, e.g. to add a unit conversion or define a new trait. This should be done on the master branch (via pull or rebase)
